client_it_test.go

```diff
@@ -208,42 +207,42 @@ func TestClientEventHandlingOrder(t *testing.T) {
 	// Create custom cluster, and client from it
 	cls := it.StartNewClusterWithOptions("event-order-test-cluster", 15701, it.MemberCount())
 	defer cls.Shutdown()
-	conf := cls.DefaultConfig()
+	cnfg := cls.DefaultConfig()
```

Comment:
How about `cfg` instead of `cnfg` just to be consistent with other tests?

Reply:
Thanks, fixed at https://github.com/hazelcast/hazelcast-go-client/pull/749/commits/3ff8826cf8dafa7914a11a5c3b203f6b6c43dd72